torch/nn/modules/container.py

```diff
@@ -621,6 +621,20 @@ def pop(self, key: str) -> 'Parameter':
         del self[key]
         return v
 
+    def get(self, key: str, default: Optional['Parameter'] = None) -> 'Parameter | None':
+        r"""Return the parameter associated with key if present.
+            Otherwise return default if provided, None if not.
+
+        Args:
+            key (string): key to get from the ParameterDict
+            default (Parameter, optional): value to return if key not present
+        """
+        return self._parameters.get(key, default)
+
+    def fromkeys(self, keys: Iterable['str'], default: Optional['Parameter'] = None) -> 'ParameterDict':
```

Comment:
This might need a @classmethod annotation. I'm p sure you should be able to use this as a class method. I.e. you can call ParameterDict.fromkeys(). This would also mean that you would need to update line 635 to be OrderedDict.fromKeys().

Reply:
I think this is actually an object method? From what docs I've been able to find, at least, it seems like you call it on a specific dictionary.